python/nammu/controller/NammuController.py

```diff
@@ -8,21 +8,20 @@
 @author: raquel-ucl
 '''
 
-import codecs, os
+import codecs, os, logging
 from java.lang import System, Integer
 from javax.swing import JFileChooser, JOptionPane, ToolTipManager
 from javax.swing.filechooser import FileNameExtensionFilter
-
-from pyoracc.atf.atffile import AtfFile
-from requests.exceptions import Timeout, ConnectionError, HTTPError,\
+from logging import StreamHandler, Formatter
+from logging.handlers import RotatingFileHandler
+from requests.exceptions import Timeout, ConnectionError, HTTPError, \
```

Comment:
suggest just adding a new line 
`from requests.exceptions import RequestException`
to avoid the \